app/views/users/show.html.erb

```diff
@@ -1,7 +1,28 @@
 <p style="color: green"><%= notice %></p>
 
 <%= render @user %>
+<%##############################################%>
```

Comment:
- [ ] [OPTIONAL] Kindly remove all unused codes/comments in order to have a clean codebase.